plugins/com.google.cloud.tools.eclipse.appengine.validation/META-INF/MANIFEST.MF

```diff
@@ -11,4 +11,12 @@ Require-Bundle: org.eclipse.core.resources,
   org.eclipse.jdt.core,
  javax.servlet;bundle-version="3.1.0";visibility:=reexport,
  javax.servlet.jsp;bundle-version="2.2.0";visibility:=reexport
-Import-Package: org.eclipse.core.runtime
+Import-Package: com.google.cloud.tools.eclipse.appengine.facets,
+ com.google.common.annotations,
+ com.google.common.base,
+ com.google.common.collect,
+ com.google.common.io,
+ org.eclipse.core.databinding.validation,
+ org.eclipse.core.runtime,
```

Comment:
I think you can remove it (as long as it doesn't increase the current count of warnings).